ratenyu/courses/course_util.py

```diff
@@ -66,6 +67,14 @@ def save_new_review(
     )
     course_id = course_id_query(course_subject_code, catalog_number).course_id
     class_obj = get_class(course_id=course_id, professor_name=professor_name)
+    existing_review = Review.objects.filter(user=user)
```

Comment:
existing_review = Review.objects.filter(user=request.user, class_id=get_class(course_id, professor_name))
I think we can use this.. We will refrain from using the loop.

Reply:
The problem is we want to only let the user post 1 review per course, not one review per class. This filter only looks for if you've written a review for that professor + that course.

Reply:
as an example this would let someone post multiple reviews for a given course as long as they choose a different professor each time